.github/workflows/manage-issue-header.yml

```
@@ -1,47 +1,46 @@
 name: Manage issue header
 
 on:
-    issues:
-      types: [labeled, unlabeled]
-    workflow_call:
-      secrets:
-        LE_BOT_APP_ID:
-            description: "GitHub App ID for authentication"
-            required: true
-        LE_BOT_PRIVATE_KEY:
-            description: "GitHub App Private Key for authentication"
-            required: true
+  workflow_call:
+    secrets:
+      LE_BOT_APP_ID:
+          description: "GitHub App ID for authentication"
+          required: true
+      LE_BOT_PRIVATE_KEY:
+          description: "GitHub App Private Key for authentication"
+          required: true
+
 jobs:
   manage-issue-header:
     runs-on: ubuntu-latest
     if: |
+      github.event.action == 'opened' ||
       (github.event.action == 'labeled' && github.event.label.name == 'help wanted') ||
       (github.event.action == 'unlabeled' && github.event.label.name == 'help wanted')
     steps:
-      - name: Generate App Token
+      - name: Generate GitHub token
         id: generate-token
         uses: tibdex/github-app-token@v2
         with:
           app_id: ${{ secrets.LE_BOT_APP_ID }}
           private_key: ${{ secrets.LE_BOT_PRIVATE_KEY }}
 
-      - name: Checkout called repository
+      - name: Checkout .github repository
         uses: actions/checkout@v3
         with:
-          repository: learningequality/.github
-          ref: main
+          repository: MisRob/.github
```

@MisRob TODO before merge

```suggestion
          repository: learningequality/.github
```